applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -603,6 +612,25 @@ Selecting a configuration
 Before you start testing the application, you can select one of the :ref:`matter_bridge_app_custom_configs`.
 See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information how to select a configuration.
 
+.. _configuration_matter_bridge_functionality:
```

Comment:
I'm not 100% if this is important or not, but usually all ref links within the same file starts with the same prefix, so I would recommend the following change. Also please remember to update the name above (in the place where it's used as a link).
```suggestion
.. _matter_bridge_smart_plug_functionality:
```